base/server/python/pki/server/cli/__init__.py

```diff
@@ -482,3 +483,63 @@ def execute(self, argv):
             sys.exit(1)
 
         instance.restart()
+
+
+class RunCLI(pki.cli.CLI):
+
+    def __init__(self):
+        super(RunCLI, self).__init__('run', 'Run PKI server in foreground')
+
+    def print_help(self):
+        print('Usage: pki-server run [OPTIONS] [<instance ID>]')
+        print()
+        print('      --as-current-user         Run as current user.')
+        print('  -v, --verbose                 Run in verbose mode.')
```

Comment:
Going with the earlier suggestion about `--debug` potentially invoking `jdb`, what if we change `--verbose` and `--debug` to a `--level=<VERBOSE,...>`? Would that be possible under `getopt` or would that require you to switch to `argparse`? 

Just a thought to make development easier and to clarify arguments.

Reply:
Both `--verbose` and `--debug` have been used quite consistently everywhere else, so I'd like to keep it that way.

Reply:
BTW, by default the CLIs run with WARN level, but you can specify `--verbose` or `--debug` to increase the debug level if necessary. Usually these are sufficient.

Reply:
Fair enough. Then I think that leaves passing additional options.